content/developers/metrics/count.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Count
+kind: documentation
+further_reading:
+- link: "developers/metrics"
```

Comment:
Should we add more further reading links? like to `/developers/agent_check` for instance?

Reply:
All of the Further Reading lists need to be improved. We can iterate after.